lusidtools/cocoon/utilities.py

```diff
@@ -1327,3 +1327,38 @@ def update_value(d: typing.Union[dict, str], val: typing.Union[str, float]):
     d = val
 
     return d
+
+
+def enrich_existing_column(data_frame, mapping, field_name, enriched_column, constant_prefix):
+    # A column for name already exists and needs to be enriched
```

Comment:
Can you please add type hints, a docstring and add the @checkargs decorator now that you've moved this to a function?

Reply:
Added:
- type hints
- @checkargs
- numpy style docstring